ir/src/main/scala/org/scalajs/core/ir/Serializers.scala

```diff
@@ -1040,19 +939,13 @@ object Serializers {
     }
 
     def readJSNativeLoadSpec(): Option[JSNativeLoadSpec] = {
-      if (useHacks068) {
-        Some(readString()).filter(_ != "").map { jsFullName =>
-          JSNativeLoadSpec.Global(jsFullName.split("\\.").toList)
-        }
-      } else {
-        (input.readByte(): @switch) match {
-          case TagJSNativeLoadSpecNone =>
-            None
-          case TagJSNativeLoadSpecGlobal =>
-            Some(JSNativeLoadSpec.Global(readStrings()))
-          case TagJSNativeLoadSpecImport =>
-            Some(JSNativeLoadSpec.Import(readString(), readStrings()))
-        }
+      (input.readByte(): @switch) match {
+        case TagJSNativeLoadSpecNone =>
+          None
+        case TagJSNativeLoadSpecGlobal =>
+          Some(JSNativeLoadSpec.Global(readStrings()))
+        case TagJSNativeLoadSpecImport =>
+          Some(JSNativeLoadSpec.Import(readString(), readStrings()))
```

Comment:
Why would those need newlines between them? They don't have blank lines inside their cases, so it's not required to have a blank line between the cases either.